graphql/client/client.go

```diff
@@ -50,3 +55,33 @@ func Services(cliCtx *cli.Context, duration schema.Duration) []schema.Service {
 
 	return response["services"]
 }
+
+func Instances(cliCtx *cli.Context, serviceId string, duration schema.Duration) []schema.ServiceInstance {
+	client := newGraphqlClient(cliCtx)
+
+	var response map[string][]schema.ServiceInstance
+	request := graphql.NewRequest(`
+		query ($serviceId: ID!, $duration: Duration!) {
+    		instances: getServiceInstances(duration: $duration, serviceId: $serviceId) {
+      			id
+      			name
+      			language
+				instanceUUID
+      			attributes {
+        			name
+        			value
+      			}
+			}
+		}
+	`)
+	request.Var("serviceId", serviceId)
+	request.Var("duration", duration)
+
+	ctx := context.Background()
+	if err := client.Run(ctx, request, &response); err != nil {
+		logger.Log.Fatalln(err)
+		panic(err)
```

Comment:
Seems we don't need `panic`, `logger.Log.Fatalln(err)` calls `os.exit`, and `panic` is unreachable

Reply:
I'll check all the code

Reply:
done